gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/ConditionalPatchCheck.java

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
is this whole concept needed because something verifies that if you reference the name of a check in some configuration but that check can't be found on the classpath it throws an error?  I'm sketchy on the details of how checks are found/activated in our tooling so this may be a dumb question.

Reply:
For the safe `PreferSafeLogger` check for example, we only want to actually do automated refactoring from `Logger` to `SafeLogger` when the `com.palantir.safe-logger:logger` jar is on the classpath. We [already do this in baseline](https://github.com/palantir/gradle-baseline/blob/4c71b30d2ba8a81cfb63cba8bd163207188c82a3/gradle-baseline-java/src/main/java/com/palantir/baseline/plugins/BaselineErrorProne.java#L260-L321), imo [it's much nicer with this construct](https://github.com/palantir/gradle-baseline/blob/63b5336f9767a91b9dedcc4ccc5214533dcf0722/gradle-baseline-java/src/main/java/com/palantir/baseline/plugins/BaselineErrorProne.java#L63-L64).